lib/resilient/circuit_breaker/registry.rb

```diff
@@ -0,0 +1,33 @@
+module Resilient
+  class CircuitBreaker
+    class Registry
```

Comment:
Yep. Makes room for a thread safe version down the road or whatever and the interface stays small and explicit.